docs/config.qmd

```diff
@@ -0,0 +1,68 @@
+---
+title: "Configuration"
+editor:
+  markdown:
+    wrap: sentence
+    canonical: true
+---
+
+Air can be configured using a TOML configuration file named `air.toml`.
+This file should be placed in your project's root directory.
```

Comment:
I think the real story is probably a little more complicated. I poked around PRs a little bit but wasn't completely confident.

If there is more to the story, you might want to include a "File location" section in this doc (or just revise this sentence).

Reply:
Currently it can be at the root directory _or above it_, so if your root is `files/dplyr/` then we definitely recommend you put an `air.toml` in `files/dplyr`, but if it's in `files/` we will find that too and use it.

In the long term we also want to look _down_ into the project directory too. So if you open `files/dplyr` in Positron but have this structure:

```
files/dplyr/air.toml
files/dplyr/R/air.toml
```

and you reformat a file in `files/dplyr/R`, then we want it to use `files/dplyr/R/air.toml`, which it currently won't do

---

I can take over the notes for this though, thanks!